airbyte-webapp/src/hooks/useRouter.tsx

```diff
@@ -1,39 +1,19 @@
 import queryString from "query-string";
```

Comment:
Could we maybe rename this file to `useQuery` now, since it does not contain any hook named `useRouter` anymore.